docs/framework/whats-new/obsolete-members.md

```diff
@@ -318,7 +318,7 @@ The tables in this article list the type members that are obsolete in the
 |----------|------------|-------------|  
 |<xref:System.Activities.Presentation.DragDropHelper?displayProperty=fullName>|<xref:System.Activities.Presentation.DragDropHelper.DoDragMove%28System.Activities.Presentation.WorkflowViewElement%2CSystem.Windows.Point%29>|First deprecated in the .NET Framework 4.5.<br /><br /> This method does not support dragging multiple items. Use                                          <!--zz <xref:System.Activities.Presentation.DragDropHelper.DoDragMove%28System.Collections.Generic.IEnumerable%7BSystem.Activities.Presentation.WorkflowViewElement%7D%2CSystem.Windows.Point%29> -->   `System.Activities.Presentation.DragDropHelper.DoDragMove` instead.|  
```

Comment:
The obsolete text should read, "This method does not support dragging multiple items."